test/webpack-build.test.js

```diff
@@ -193,18 +193,37 @@ test('Bundles can use custom names', async (test) => {
   test.true(await fs.pathExists(path.join(test.context.buildDirectory, 'lambda', 'service.js')));
 });
 
-test('Multi-entry bundling cannot be used with bundle zipping', async (test) => {
-  const options = {
+test('Bundles for multiple entries can be zipped', async (test) => {
+  await build({
     entrypoint: [
-      path.join(__dirname, 'fixtures', 'lambda_service.js'),
-      path.join(__dirname, 'fixtures', 'lambda_graphql.js')
+      path.join(__dirname, 'fixtures', 'lambda_graphql.js:graphql.js'),
+      path.join(__dirname, 'fixtures', 'lambda_service.js:lambda/service.js')
+    ],
+    outputPath: test.context.buildDirectory,
+    serviceName: 'test-service',
+    zip: true
+  });
+
+  test.true(await fs.pathExists(path.join(test.context.buildDirectory, 'graphql.js.zip')));
+  test.true(await fs.pathExists(path.join(test.context.buildDirectory, 'lambda/service.js.zip')));
+});
+
+test('Expand input entrypoint directory into multiple entrypoints', async (test) => {
+  await build({
+    entrypoint: [
+      path.join(__dirname, 'fixtures/multi-lambdas')
     ],
     outputPath: test.context.buildDirectory,
     serviceName: 'test-service',
     zip: true
-  };
+  });
+
+  await fs.mkdirp(path.join(__dirname, 'fixtures/multi-lambdas/empty'));
 
-  await test.throws(build(options), /multiple entrypoints/i);
+  test.true(await fs.pathExists(path.join(test.context.buildDirectory, 'func1.js.zip')));
```

Comment:
Stripping the directory from the entries on output seems potentially unsafe. What happens if something like `lambda-tool-build -o buildDirectory ./func1.js ./fixtures/multi-lambdas` is done? It seems to me that the contents of `func1.js.zip` would not be well-defined for the consumer in this case.

Reply:
I don't think a change is necessary. I think the developer should be responsible for not producing lambda functions with the same name. I think `webpack` might detect that output files would be overridden as well. Also, I think you can produce two lambda output files with the same name before my changes.

Reply:
Yes, you could do something like `src/foo.js:bar.js src/baz.js:bar.js` but I would agree in this case that it is the consumers fault. Without overriding the default naming it is not possible to create collisions since the file system will guarantee uniqueness. However, the case I mentioned is not as straight forward or obvious since the default behavior can create collisions without warning. My opinion is that if you provide a directory as an entry point then the output should be qualified with the same directory name by default so that the default "just works".

A nice enhancement would be to always warn on collisions, but that is outside the scope of this PR.